Maddori.Apple/Maddori.Apple/Screen/Setup/JoinTeam/JoinTeamViewController.swift

```diff
@@ -69,11 +77,31 @@ final class JoinTeamViewController: BaseTextFieldViewController {
         view.buttonAction = { [weak self] in
             self?.presentCreateTeamViewController()
         }
+        view.isHidden = fromView == .loginView ? false : true
         return view
     }()
+    private lazy var closeButton: CloseButton = {
+        let button = CloseButton()
+        let action = UIAction { [weak self] _ in
+            self?.dismiss(animated: true)
+        }
+        button.addAction(action, for: .touchUpInside)
+        button.isHidden = fromView == .teamManageView ? false : true
+        return button
+    }()
     
     // MARK: - life cycle
     
+    init(from: ViewType) {
+        self.fromView = from
+        super.init()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
p2;

요기도 마찬가지로
```suggestion
    @available(*, unavailable)
    required init?(coder: NSCoder) {
        nil
    }
```

로 가능할 것 같습니다!

Reply:
[561f6f9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/328/commits/561f6f9b1a4a93b5fe611118433fa737418a4e15)

완료!